app/src/main/java/com/example/myapplication/MaterialActivity.kt

```diff
@@ -0,0 +1,91 @@
+package com.example.myapplication
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.text.Editable
+import android.text.TextWatcher
+import android.widget.*
+import kotlinx.android.synthetic.main.activity_material.*
+import java.util.regex.Pattern
+
+class MaterialActivity : AppCompatActivity() {
+    var pattern = Pattern.compile(".+@.+\\.[a-z]+")
```

Comment:
proper spacing and make private if required remove commented code and naming not should be `bool`